charts/rancher-webhook/Chart.yaml

```diff
@@ -9,6 +9,7 @@ annotations:
   catalog.cattle.io/namespace: cattle-system
   catalog.cattle.io/release-name: rancher-webhook
   catalog.cattle.io/os: linux
+  catalog.cattle.io/rancher-version: '>2.6.0-0'
```

Comment:
Should this be `>2.6.0-alpha`?

Reply:
'>2.6.0-0' is lower than `2.6.0-alpha`, and we decide to go with `2.6.0-0` for all charts. 
See the comparison at https://semvercompare.azurewebsites.net/?version=2.6.0-1&version=2.6.0-alpha&version=2.6.0-0&version=2.6.0-beta